core-bundle/src/Security/User/ContaoUserProvider.php

```diff
@@ -115,8 +114,8 @@ private function validateSessionLifetime(User $user): void
             return;
         }
 
-        $config = $this->framework->getAdapter(Config::class);
-        $timeout = (int) $config->get('sessionTimeout');
+        // TODO: where should we get this value from?
+        $timeout = (int) \ini_get('session.gc_maxlifetime');
 
         if ($timeout > 0 && time() - $this->session->getMetadataBag()->getLastUsed() < $timeout) {
             return;
```

Comment:
/cc @bytehead

Reply:
Symfony uses `ini_get('session.cookie_lifetime')` and also `framework.session.cookie_lifetime`.

Reply:
Do we even need our own timeout?

Reply:
That's not correct. You should take it from the configured session service. If I don't use the native implementation, the ini settings are completely irrelevant.

Reply:
@fritzmg yes we do and it's also always recommended to use a custom one on top of the server-side configured one. Just because a session is available 7 days doesn't mean your app wants it to be valid for that long.

Reply:
But ideally they should be identical. It's good practice to double check though :)

Reply:
What I meant is: is it actually necessary to validate the session life time in our user provider. Isn't this the task of the storage? I always assumed we only did it there because of our legacy `sessionTimeout` config.

Reply:
> Symfony uses ini_get('session.cookie_lifetime') and also framework.session.cookie_lifetime.

`framework.session.cookie_lifetime` then simply gets set via `ini_set('session.cookie_lifetime', …)` by `NativeSessionStorage`.

Reply:
Done: 326b07d4396b27b8ed332c505342d5f84dc7d5e1